src/pages/editor/flow/components/EditorDetailPanel/DetailForm.tsx

```diff
@@ -0,0 +1,137 @@
+import { Form } from '@ant-design/compatible';
+import { Card, Input, Select } from 'antd';
+import React, { Fragment } from 'react';
+
+import { FormComponentProps } from 'antd/es/form';
+import { withPropsAPI } from 'gg-editor';
+
+const upperFirst = (str: string) =>
+  str.toLowerCase().replace(/( |^)[a-z]/g, (l: string) => l.toUpperCase());
+
+const { Item } = Form;
+const { Option } = Select;
+
+const inlineFormItemLayout = {
+  labelCol: {
+    sm: { span: 8 },
+  },
+  wrapperCol: {
+    sm: { span: 16 },
+  },
+};
+
+interface DetailFormProps extends FormComponentProps {
+  type: string;
+  propsAPI?: any;
+}
+
+class DetailForm extends React.Component<DetailFormProps> {
+  get item() {
+    const { propsAPI } = this.props;
+
+    return propsAPI.getSelected()[0];
+  }
+
+  handleSubmit = (e: React.FormEvent) => {
+    if (e && e.preventDefault) {
+      e.preventDefault();
+    }
+
+    const { form, propsAPI } = this.props;
+    const { getSelected, executeCommand, update } = propsAPI;
+
+    setTimeout(() => {
+      form.validateFieldsAndScroll((err, values) => {
+        if (err) {
+          return;
+        }
+
+        const item = getSelected()[0];
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['magic numbers' are not allowed: 0](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)